### PR TITLE
Add support for multi-value update access commands

### PR DESCRIPTION
**Pull request by devinlundberg**
This will allow for knox key acls to be updated in bulk without running numerous commands. We can also create template acl files to make key generation easier for complex services.

### CODE REVIEW

File: client/updateaccess.go [hunk 1]
```diff
@@ -49,6 +53,26 @@ var updateAccessService = cmdUpdateAccess.Flag.Bool("S", false, "")
 var updateAccessServicePrefix = cmdUpdateAccess.Flag.Bool("N", false, "")
 
 func runUpdateAccess(cmd *Command, args []string) {
+	if *updateAccessACL != "" {
+		if len(args) != 1 {
+			fatalf("access takes exactly one arguments. See 'knox help access'")
```
Comment (csstaub) [inline client/updateaccess.go:58]: This sounds wrong to me, can't it also take more than one arg based on situation?

File: client.go [hunk 2]
```diff
@@ -160,7 +160,7 @@ type APIClient interface {
 	GetKeys(keys map[string]string) ([]string, error)
 	DeleteKey(keyID string) error
 	GetACL(keyID string) (*ACL, error)
-	PutAccess(keyID string, a *Access) error
+	PutAccess(keyID string, acl ...Access) error
```
Comment (csstaub) [inline client.go:163]: Do we have a test to make sure that old knox client binaries won't break?

Comment (devinlundberg) [reply client.go:163]: yes there is a server side test in routes_test.go to ensure that the old apis still work with the access parameter. I'll add a test for the new acl parameter there as well (so it is clear that both are tested).

I also manually tested with dev_client and dev_server and the old and new clients both work.

File: client/updateaccess.go (hunk 1 above)
Comment (krockpot) [reply client/updateaccess.go:58]: it can take more than one arg if you *don't* specify the --acl flag (which is the check on line 56)

Comment (devinlundberg) [reply client/updateaccess.go]: I fixed the wording to say it takes one arg when --acl is used

**Pull request merged**